src/server/rpc/procedures/battleship/battleship.js

```diff
@@ -63,6 +71,14 @@ Battleship.prototype.start = function() {
     return true;
 };
 
+/**
+ * Place a ship on the board
+ * @param {String} ship Ship type to place
+ * @param {BoundedNumber<1,100>} row Row to place ship in
+ * @param {BoundedNumber<1,100>} column Column to place ship in
+ * @param {BoundedNumber<0,3>} facing Direction to face
+ * @returns {Boolean} If piece was placed
+ */
 Battleship.prototype.placeShip = function(ship, row, column, facing) {
```

Comment:
It looks like the annotation is incorrect here for the `facing` argument (it is expecting a string for the direction - check out the Battleship example). This is currently breaking the Battleship example.